packages/spor-react/src/theme/components/datepicker.ts

```diff
@@ -109,9 +122,19 @@ const config = helpers.defineMultiStyleConfig({
         }),
       },
     },
+    arrow: {
+      [$arrowBackground.variable]: mode("white", colors.night)(props)
+    },
+    calendar: {
+      backgroundColor: mode("white", "night")(props),
+      color: mode("darkGrey", "white")(props)
+    },
+    weekDays: {
+      color: props.index < 5 ? mode("darkGrey", "white")(props) : mode("greenHaze", "azure")(props)
```

Comment:
Her vettu - unngå å sende inn index her direkte - bruk enten en CSS selektor eller lag en egen greie for helgene.

Her er en CSS-selector som gjør det samme som index f.eks.:

```tsx
weekDays: {
  color: mode("greenHaze", "azure")(props),
  ":nth-child(5 + n) &": {
    color: mode("darkGrey", "white")(props)
  }
}
```